spec/order_summary_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'order_summary'
+
+describe OrderSummary do
+
+  subject(:order_summary) { described_class.new }
+  # let(:order) { double('order') }
+
+  context 'order summary' do
+
+    it 'contains a log of the order' do
+      # allow(order).to receive(:add_item).with("Golden French Fries") { {"Golden French Fries" => 1} }
```

Comment:
Metrics/LineLength: Line is too long. [103/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)